runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/Scheduler.java

```diff
@@ -39,8 +40,13 @@
 @Service
 public class Scheduler {
   private static class FinishedWork {
+    Instant created = Instant.now();
     NotificationDto dto;
     int epoch;
+
+    public boolean shouldRerun() {
+      return !dto.getHealthType().isDone() && Duration.between(created, Instant.now()).toMinutes() < 10;
```

Comment:
Is this necessary with the scheduler hard-coded to scan every 15 minutes? If so, should this be greater than instead of less than?